src/Sulu/Bundle/MediaBundle/Media/FileValidator/DefaultFileValidator.php

```diff
@@ -57,7 +57,7 @@ public function validate (UploadedFile $file, $methods = array(
             throw new InvalidFileTypeException('The file type was blocked');
         }
 
-        if (in_array(self::VALIDATOR_MAX_FILE_SIZE, $methods) && $file->getSize() <= $this->maxFileSize) {
+        if (in_array(self::VALIDATOR_MAX_FILE_SIZE, $methods) && $this->maxFileSize !== null && $file->getSize() >= $this->maxFileSize) {
             throw new MaxFileSizeExceededException('The file is to big');
```

Comment:
there was/is a failing test in ContactBundle for import, @christianbader had a look at it earlier